aiozk/session.py

```diff
@@ -84,6 +84,8 @@ def __init__(self, servers, timeout, retry_policy, allow_read_only):
                 conn = await self.make_connection(host, port)
                 if not conn or (conn.start_read_only and not allow_read_only):
                     continue
+                else:
```

Comment:
When you pass multi servers into Client class, for example servers="zk1,zk2,zk3", then you will make one connection to each server, but only the last connection you will hold in self.conn, the others are leaking. I am quiting the loop when I get the first available connection.